### DISCUSSION TITLE
包你们能打开打赏本站吗？我怎么打不开？

### DISCUSSION THREAD

**Discussion by ghost** [Gamuxorg/bbs#328] [游戏或网站运行反馈]
### 以下是推荐模板
游戏名称：

https://www.linuxgame.cn/
操作系统：*（uname -a）*火狐浏览器

bug描述：*（具体说说发生了什么）*点击打赏本站之后没有反应

终端运行输出 ：*（如果有的话）*

自行尝试解决bug的方式和结果：*（如果有的话）*我没用国内节点测试

**Comment by shouhuanxiaoji** [comment]
忘记添加了。。

**Comment by LTreeshu** [comment]
我也打不开，快添加一个吧

**Comment by TailorSchmitt** [comment]
确实，单击了“打赏本站”后没有任何反应。建议赶快添加。我相信很多人都会愿意打赏的，我作为一个学生党，也想捐一点点钱以表心意。2020年度财务状况确实令人担忧。用户也不希望这个站点亏损，毕竟大家都期盼站点能持续运行很长时间。

**Comment by shouhuanxiaoji** [comment]
啊，行行行，好好好
今年是本站运营第11年了，不用担忧资金问题导致的持续运行不下去的问题。除非linux用户不再需要我们

**Comment by shouhuanxiaoji** [comment]
已添加
